autometa/taxonomy/ncbi.py

```diff
@@ -217,10 +222,13 @@ def rank(self, taxid):
         str
             rank name if taxid is found in nodes.dmp or merged.dmp else 'unclassified'
         """
-        self.is_valid_taxid(taxid)
-        rank_name = self.nodes.get(taxid, {'rank': 'unclassified'}).get('rank')
-        if rank_name == {'rank': 'unclassified'}:
-            return self.merged.get(taxid, {'rank': 'unclassified'}).get('rank')
+        if not self.is_valid_taxid(taxid):
+            return False
```

Comment:
Do you want to return False here? I thought the function was supposed to return `'unclassified'` if it could not find a rank?

Reply:
True, I was returning `False` based on https://github.com/KwanLab/Autometa/pull/83#discussion_r426834107 , https://github.com/KwanLab/Autometa/pull/83#discussion_r426817167 and https://github.com/KwanLab/Autometa/pull/83#discussion_r426822814
Please correct me if I'm wrong but the way I understood it was that we will always get an input of `taxid` that is present in the database and thus we won't need `unclassified`

Reply:
I don't see how returning `False` here addresses the comments made by Jason. It seems I have made a similar comment as him. I.e. what should each method be doing?